report_py3o/models/py3o_template.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 XCG Consulting (http://odoo.consulting)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import fields, models
+
+
+class Py3oTemplate(models.Model):
+    _name = 'py3o.template'
+
+    name = fields.Char(required=True)
+    py3o_template_data = fields.Binary("LibreOffice template")
```

Comment:
@lasley py3o_template refers to the file used as template... IMO the name appropriate